fetch.bs

```diff
@@ -7168,6 +7168,24 @@ returns failure or a <a for=/>MIME type</a>.
 
 <hr>
 
+<div algorithm>
+<p>To <dfn id=concept-body-get-mime-type for=Body>get the MIME type</dfn>, given an object, <var>requestOrResponse</var>, including the
+{{Body}} interface mixin, run these steps:
+
+<ol>
+ <li><p>If <var>requestOrResponse</var>'s has an associated <a for=Headers>header list</a>,
+ <var>headers</var>, and it's non-null, then:
+ <ol>
+   <li><p>Let <var>mimeTypeInHeaders</var> be the result of <a for="header list">extracting a
+   MIME type</a> from <var>headers</var>.
+   <li><p>If <var>mimeTypeInHeaders</var> is non-null, then return <var>mimeTypeInHeaders</var>.
+ </ol>
+ <li><p>Let <var>bodyWithType</var> be the result of <a for=BodyInit>extracting</a> <a>this</a>'s
+ <a for=Body>body</a>.
+ <li><p>Return <var>bodyWithType</var>'s <a for="body with type">type</a>.
```

Comment:
This shouldn't be needed. This already happens as part of constructing a Request or Response, no?

Reply:
Indeed! You mean, while given a Request or Response object, it must already run through the "extract" algorithm (e.g. Step 37-1 in https://fetch.spec.whatwg.org/#dom-request) and set body's type back to its headers so we don't need to "extract" again. Is my understanding correct?

Reply:
Yes, `Content-Type` will already be in the header list (or not).